sbt/src/sbt-test/dependency-management/cached-resolution-circular/changes/multi.sbt

```diff
@@ -0,0 +1,57 @@
+lazy val check = taskKey[Unit]("Runs the check")
+
+val sprayV = "1.1.1"
+val playVersion = "2.2.0"
+val summingbirdVersion = "0.4.0"
+val luceneVersion = "4.0.0"
+val akkaVersion = "2.3.1"
+
+def commonSettings: Seq[Def.Setting[_]] =
+  Seq(
+    ivyPaths := new IvyPaths( (baseDirectory in ThisBuild).value, Some((target in LocalRootProject).value / "ivy-cache")),
+    scalaVersion := "2.10.4",
+    fullResolvers := fullResolvers.value.filterNot(_.name == "inter-project"),
+    updateOptions := updateOptions.value.withCachedResolution(true)
+  )
+
+lazy val a = project.
+  settings(commonSettings: _*).
+  settings(
+    name := "a",
+    libraryDependencies := Seq(
+      organization.value %% "c" % version.value,
+      "commons-io" % "commons-io" % "1.3",
+      "org.apache.spark" %% "spark-core" % "0.9.0-incubating",
+      "org.apache.avro" % "avro" % "1.7.7",
+      "com.linkedin.pegasus" % "data-avro" % "1.9.40",
+      "org.jboss.netty" % "netty" % "3.2.0.Final"
+    )
+  )
+
+lazy val b = project.
+  settings(commonSettings: _*).
+  settings(
+    name := "b",
+    // this adds circular dependency
```

Comment:
I guess it's surprising to me that it would be inevitable to have a circle.  I think that hits at some other problem, but if we're handling it now already, it'd be good to fix it.